.changeset/cold-cooks-battle.md

```diff
@@ -0,0 +1,21 @@
+---
+'@astrojs/node': minor
+---
+
+Allows disabling Astro's default [HTML streaming](https://docs.astro.build/en/guides/on-demand-rendering/#html-streaming) for pages rendered on demand
```

Comment:
```suggestion
Adds a new experimental configuration option `experimentalDisableStreaming` to allow you to opt out of Astro's default [HTML streaming](https://docs.astro.build/en/guides/on-demand-rendering/#html-streaming) for pages rendered on demand.
```
For the changeset, we should introduce what the new thing is! (The above is perfect as a definition reference entry when people already know what they are looking at, and we're explaining what it is. Subtle difference when we have to announce that this new thing actually exists!)